### PR TITLE
Update Korean glossary for l10n

### PR DESCRIPTION
**Pull request by gochist**
Update Korean glossary

### CODE REVIEW

File: content/ko/docs/contribute/localization_ko.md [hunk 1]
```diff
@@ -279,8 +286,12 @@ Pull Request | 풀 리퀘스트 |
 Pull Secret Credentials | 풀(Pull) 시크릿 자격증명 |
 QoS Class | QoS 클래스 |
 Quota | 쿼터 |
+readiness gate | 준비성 게이트(rediness gate) |
```
Comment (sftim) [inline content/ko/docs/contribute/localization_ko.md]: nit:
```suggestion
readiness gate | 준비성 게이트(readiness gate) |
```

**Comment by ysyukr** [comment]
@sftim Thank you!

/lgtm

**Pull request merged**